contracts/interface/IAccountBalance.sol

```diff
@@ -232,4 +239,9 @@ interface IAccountBalance {
         address baseToken,
         int256 accountValue
     ) external view returns (int256);
+
```

Comment:
L221 should be updated as well.

`getTotalPositionValue `'s comment
`/// @dev we use `15 mins` twap to calc position value`

Reply:
Fixed in 01ba903afff31d0aef7bb7f137760ef0241f59fc